src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  let CURRENT_STATE = { ...state };
```

Comment:
it's not a constant value like (40 or 50), please use `camelCase` 
```suggestion
  const CURRENT_STATE = { ...state };
```